_posts/2021-12-27-partner-engineering-the-missing-manual.md

```diff
@@ -54,7 +54,7 @@ All this talk about integrations – come to think of it, *what is an integratio
 
 In general, integrations fall into two main categories and in both cases serve to improve the product:
 1. **🧩 Complementary**, where a product *requires* or is *made complete* by external integrations. Smart TVs need partnerships with streaming providers, app stores need apps, etc.
-2. **🪄 Supplementary**, where a product is *reinforced* or *enhanced* by external integrations. Video calling software might be better with external apps but works just fine without it.
+2. **🧪 Supplementary**, where a product is *reinforced* or *enhanced* by external integrations. Video calling software might be better with external apps but works just fine without it.
```

Comment:
How strange! The magic wand emoji (🪄) renders file on my Chrome macOS – let's trying updating to nail care (💅).

```suggestion
2. **💅 Supplementary**, where a product is *reinforced* or *enhanced* by external integrations. Video calling software might be better with external apps but works just fine without it.
```